libs/libarchfpga/src/read_xml_arch_file.cpp

```diff
@@ -1675,13 +1677,35 @@ static void ProcessMode(pugi::xml_node Parent, t_mode * mode,
 	/* Allocate power structure */
 	mode->mode_power = (t_mode_power*) vtr::calloc(1, sizeof(t_mode_power));
 
+	mode->meta = ProcessMetadata(Parent, loc_data);
+
 	/* Clear STL map used for duplicate checks */
 	pb_type_names.clear();
 
 	Cur = get_single_child(Parent, "interconnect", loc_data);
 	ProcessInterconnect(Cur, mode, loc_data);
 }
 
+static t_metadata_dict ProcessMetadata(pugi::xml_node Parent,
+		const pugiutil::loc_data& loc_data) {
+	//	<metadata>
+	//	  <meta x_offset="12" y_offset="100" name="grid_prefix">CLBLL_L_</meta>
```

Comment:
Should update the comment to remove x_offset, y_offset example as it isn't in the metadata anymore.